syft/messaging/message.py

```diff
@@ -300,7 +300,7 @@ def __init__(self, obj_id, user, reason):
 
     def __str__(self):
         """Return a human readable version of this message"""
-        return f"({type(self).__name__} {(self.obj_id, self.user, self.reason)})"
+        return f"({type(self).__name__} {(self.obj_id if hasattr(self, 'obj_id') else '', self.user, self.reason)})"
```

Comment:
This bypasses the error, but I think the underlying ssue is that the field on the message is called `object_id` (instead of `obj_id`.)